src/wp-content/themes/twentytwentyone/assets/sass/04-elements/blockquote.scss

```diff
@@ -2,6 +2,12 @@ blockquote {
 	padding: 0;
 	position: relative;
 	margin: var(--global--spacing-vertical) 0 var(--global--spacing-vertical) var(--global--spacing-horizontal);
+	letter-spacing: var(--heading--letter-spacing-h4);
```

Comment:
I'm not really sure why the letter spacing is used here in the blockquote element, but not on the _block_. I can't recall. Maybe it was only intended for the classic editor quote.